internal/services/cdn/frontdoorruleconditions/cdn_frontdoor_rule_conditions.go

```diff
@@ -231,55 +232,56 @@ func validateCdnFrontDoorExpandConditionOperatorValues(operator string, matchVal
 		return fmt.Errorf("%q is invalid: no 'operator' value has been set, got %q", m.ConfigName, operator)
 	}
 
-	if operator == string(cdn.OperatorAny) && len(*matchValues) > 0 {
+	// NOTE: There are now 14 different "Any" operators in the new API, however they are all the same so I am just hardcoding this here now...
```

Comment:
Can you open an issue detailing this so it's tracked somewhere along with what we're planning to change it to if this is just for now?

Reply:
I don't believe an issue is required here, these fields were added due to the swagger issue opened by HashiCorp to allow the Pandora SDK to build the API correctly. Every `condition` now has it's own `Any` field (e.g., `ClientPortOperatorAny`, `CookiesOperatorAny`, `HostNameOperatorAny`, etc.) So I believe hardcoding this value is acceptable here as this function is a general function that will be called by all `conditions`.